src/telegram.js

```diff
@@ -448,7 +448,7 @@ class TelegramBot extends EventEmitter {
    */
   sendChatAction(chatId, action) {
     const form = {
-      action,
+      action: action,
```

Comment:
This is valid ES6 syntax. Please see https://github.com/lukehoban/es6features#enhanced-object-literals for more information on **shorthand assignments**.